### PR TITLE
[SDK-2390] Add Support for Organizations

### PR DESCRIPTION
**Pull request by frederikprijck**
This PR updates the readme to show how organizations can be used.
As the underlying SPA-JS SDK already supports this, there is no code change needed.

~~All we are waiting for is updating the SPA-JS version once it has been released, to ensure we have first-class support.~~ 
Added the latest version of SPAJS

This PR also adds basic support in our playground.

### CODE REVIEW

**Comment by stevehobbsdev** [comment]
Should we also add `organization` and `invitation` to the `AuthConfig` definitions, so that they're typed and can appear in generated docs?

**Comment by frederikprijck** [comment]
@stevehobbsdev Updated the PR to include these properties in `AuthConfig`.

File: projects/auth0-angular/src/lib/auth.service.spec.ts [hunk 1]
```diff
@@ -369,6 +369,30 @@ describe('AuthService', () => {
...
+      );
+
+      const localService = createService();
+
+      loaded(localService).subscribe(() => {
+        expect(navigator.navigateByUrl).toHaveBeenCalledWith('/');
+        done();
+      });
+    });
+
```
Comment (frederikprijck) [inline projects/auth0-angular/src/lib/auth.service.spec.ts:395]: These changes are unrelated, but I wanted to bump coverage to fix the build.

**Review by stevehobbsdev — APPROVED** [review]
🎉

**Pull request merged**